src/ec/packages/ec-component-footer/demo/data.js

```diff
@@ -1,217 +1,61 @@
 /* eslint-disable import/no-extraneous-dependencies */
+import specDataCorporate from '@ecl/ec-specs-footer/demo/data--corporate';
+import specDataCustom from '@ecl/ec-specs-footer/demo/data--custom';
+
+function formatLink(l) {
+  const link = {
+    link: {
+      label: l.label,
+      path: l.href,
+    },
+  };
+
+  if (l.icon) {
+    link.link.icon_position = l.iconPosition;
+    link.icon = {
+      path: 'static/icons.svg',
+      type: l.icon.shape.split('--')[0],
```

Comment:
Not a big deal for a demo, but the `split` operation on `l.icon.shape` could be done once, so that it's not called twice for getting information from it.

```js
const [type, name] = l.icon.shape.split.split('--')
```